### PR TITLE
DDF-2228: Makes Validation Parser Source available as a service

### PR DESCRIPTION
**Pull request by rzwiefel**
#### What does this PR do?
adds interface, implements interface. 
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@djblue 
@coyotesqrl 
@jrnorth 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@kcwire(who's absent, so in his stead) -> @stustison 
@pklinef

### CODE REVIEW

**Comment by coyotesqrl** [comment]
👍

**Comment by jrnorth** [comment]
👍

File: catalog/core/catalog-core-api/src/main/java/ddf/catalog/validation/ValidationSourcer.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package ddf.catalog.validation;
```
Comment (jrnorth) [inline catalog/core/catalog-core-api/src/main/java/ddf/catalog/validation/ValidationSourcer.java]: Needs license header

Comment (rzwiefel) [inline catalog/core/catalog-core-api/src/main/java/ddf/catalog/validation/ValidationSourcer.java]: Whoooopssss. Good catch. 
Done

**Comment by rzwiefel** [comment]
Closing because taking another route with this. Good example of a bad idea, that doesn't seem bad until you consider the architecture side.

**Pull request closed without merging**